src/SFA.DAS.EmployerAccounts/DependencyResolution/ConfigurationRegistry.cs

```diff
@@ -20,7 +19,7 @@ public ConfigurationRegistry()
             For<EmployerFinanceConfiguration>().Use(c => c.GetInstance<IAutoConfigurationService>().Get<EmployerFinanceConfiguration>(ConfigurationKeys.EmployerFinance)).Singleton();
             For<EmployerAccountsReadStoreConfiguration>().Use(c => c.GetInstance<IAutoConfigurationService>().Get<EmployerAccountsReadStoreConfiguration>(ConfigurationKeys.EmployerAccountsReadStore)).Singleton();
             For<ReferenceDataApiClientConfiguration>().Use(c => c.GetInstance<IAutoConfigurationService>().Get<ReferenceDataApiClientConfiguration>(ConfigurationKeys.ReferenceDataApiClient)).Singleton();
-            For<ReservationsClientApiConfiguration>().Use(c => c.GetInstance<IAutoConfigurationService>().Get<ReservationsClientApiConfiguration>(ConfigurationKeys.ReservationsClientApiConfiguration)).Singleton();
+            For<EmployerAccountsOuterApiConfiguration>().Use(c => c.GetInstance<IAutoConfigurationService>().Get<EmployerAccountsOuterApiConfiguration>(ConfigurationKeys.ReservationsClientApiConfiguration)).Singleton();
```

Comment:
the key being passed in here to get the config section is incorrect. `EmployerAccountsOuterApiConfiguration` should come from `c => c.GetInstance<EmployerAccountsConfiguration>().EmployerAccountsOuterApiConfiguration`

Reply:
in fact, the configuration is registered in the newly created `EmployerAccountsOuterApiRegistry` so this line should be removed.

Reply:
Done

Reply:
done